rediscache.yml

```diff
@@ -0,0 +1,77 @@
+# Description
+# ===========
+# This playbook creates Azure Cache for Redis instance, scale it up, rebooot it, add firewall rule,  delete it.
+# This playbook requires ansible >= 2.8, or role azure.azure_preview_modules.
+---
+- name: Manage Azure Cache for Redis
+  hosts: localhost
+  connection: local
+  vars:
+    resource_group: "{{ resource_group_name }}"
+    redis_name: "redis{{ resource_group_name }}"
+    location: eastus2
+
+  roles:
+    - azure.azure_preview_modules    
+
+  tasks:
+  - name: create resource group
+    azure_rm_resourcegroup:
+      name: "{{ resource_group }}"
+      location: "{{ location }}"
+
+  - name: Create Azure Cache for Redis
+    azure_rm_rediscache:
+      resource_group: "{{ resource_group }}"
+      name: "{{ redis_name }}"
+      sku:
+        name: basic
+        size: C1
+
+  - name: Wait for Redis provision succeeded
+    azure_rm_rediscache_facts:
+      resource_group: "{{ resource_group }}"
+      name: "{{ redis_name }}"
+    register: facts
+    until: "{{ facts.rediscaches[0]['provisioning_state'] == 'Succeeded' }}"
+    retries: 100
+    delay: 60
+
+  - name: Scale up Azure Cache for Redis
+    azure_rm_rediscache:
+      resource_group: "{{ resource_group }}"
+      name: "{{ redis_name }}"
+      sku:
+        name: standard
+        size: C1
+
+  - name: Wait for Redis scale ups succeeded
```

Comment:
Wait for Redis scaling up to succeed (or complete)
... i think complete sounds better